pipert2/cli.py

```diff
@@ -0,0 +1,17 @@
+import typer
```

Comment:
Maybe find a better place for this script, maybe add a new directory in utils or create a new one in the pipert2 folder itself.

Reply:
And why do we need this script ?